nx.json

```diff
@@ -1,6 +1,5 @@
 {
   "$schema": "./node_modules/nx/schemas/nx-schema.json",
-  "npmScope": "typescript-eslint",
```

Comment:
This is deprecated and no longer really used in nx, it will derive it from your existing root package